core/common/src/test/java/alluxio/ConfigurationTestUtils.java

```diff
@@ -44,11 +44,12 @@ public static InstancedConfiguration defaults() {
    * them to test specific functionality.
    *
    * @param hostname the master hostname
+   * @param journalType the master journal type
    * @param workDirectory the work directory in which to configure the journal and tiered storage
    * @return the configuration
    */
   public static Map<PropertyKey, String> testConfigurationDefaults(AlluxioConfiguration alluxioConf,
-      String hostname, String workDirectory) {
+      String hostname, String journalType, String workDirectory) {
```

Comment:
can we make the defaults independent of the journal type? We don't know at this point what the actual journal type will be, since the journal type might be overridden in the cluster configuration

Reply:
Now the `testConfigurationDefaults` is isolated from the actual journal type. The journal type is not a must-have in this method, thanks